lab1.js

```diff
@@ -54,59 +54,84 @@ assert(1 === 2, 'this is an assertion failure example. 1 === 2');
  it failed.
 */
 
-//your code goes here
+var kats = "meerkats";
+var num = 8;
+
+assert( kats === "meer" + "kats" );
+assert( kats === num, "number to string comparison returns false" );
 
 /* ----------------- Meerkats -------------------------------------------------
  Meerkats make a sort of chirping noise (according to my 30 seconds of
- research).  We're going to translate two sentences into meerkat speech.
+ research).  We"re going to translate two sentences into meerkat speech.
 */
 
-var sentence1 = 'More food please.';
-var sentence2 = 'Come over here so you can scratch my belly.';
+var sentence1 = "More food please.";
+var sentence2 = "Come over here so you can scratch my belly.";
 
 /*
  TODO: 20 points
- Your goal is to replace the words in the above sentences with 'chirp' The
- assertions at the end of this section should pass when you're done.  Use
+ Your goal is to replace the words in the above sentences with "chirp" The
+ assertions at the end of this section should pass when you"re done.  Use
  **two** different kinds of loops to implement this. (10 points each.)
- HINT: the 'split' method on String will be useful.
+ HINT: the "split" method on String will be useful.
 */
 
-//your code goes here
+var sent1arr = [];
+
+while ( sent1arr.length < 3 ) {
```

Comment:
you could use the same lead-in into this while loop that you did for the for loop. eg, split your string into an array, then use the length of that array to decide how long to run your while loop.